tensorboard/webapp/app_routing/internal_utils.ts

```diff
@@ -24,6 +25,41 @@ import {
   SerializableQueryParams,
 } from './types';
 
+/**
+ * Parses single experiments and short experiment name to id map encoded in
+ * string for card route.
+ *
+ * e.g.
+ *
+ * /card/12345/tag/tagid results,
+ * [
+ *   {name: '', id: '12345'}
+ * ]
+ *
+ * /card/exp1:12345,exp2:99999/tag/tagid results,
```

Comment:
nit: can we omit the tag part? This function is purely about parsing experiment ids